test/sdp.jl

```diff
@@ -64,6 +67,141 @@
         @test eigmin(Xv) > -1e-6
     end
 
+#    @testset "Simple SDP" begin
```

Comment:
Why keep this around?

Reply:
I have copy pasted from old/sdp.jl and started to make some changes but it is not working yet. Since changes have already been made I didn't want to loose them so I kept it commented :)

Reply:
Haha ok, carry on

Reply:
FYI the plan is to remove most of the JuMP tests that involve solving a model and checking for the answer. Instead we'll check separately that JuMP generates the model correctly and that we talk to a fake solver correctly. The JuMP tests shouldn't be used for testing the solvers.